packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -1,20 +1,48 @@
-import { css } from '@emotion/css';
+import { css, cx } from '@emotion/css';
 import React, { CSSProperties, ReactNode } from 'react';
 
-import { GrafanaTheme2 } from '@grafana/data';
+import { GrafanaTheme2, isIconName } from '@grafana/data';
 
 import { useStyles2, useTheme2 } from '../../themes';
+import { IconName } from '../../types/icon';
+import { Dropdown } from '../Dropdown/Dropdown';
+import { Icon } from '../Icon/Icon';
+import { IconButton, IconButtonVariant } from '../IconButton/IconButton';
+import { PopoverContent, Tooltip } from '../Tooltip';
+
+/**
+ * @internal
+ */
+export interface PanelChromeInfoState {
+  icon: IconName;
+  label?: string | ReactNode;
+  tooltip?: PopoverContent;
+  variant?: IconButtonVariant;
+  onClick?: () => void;
+}
```

Comment:
Not sure about the variant and onClick, think we will need more flexibility here by adding anything. make titleItems just be a React.ReactNode[]  and then have different components for different title items (one for Icon etc). The time range override is text + icon, and is not an Iconbutton